Makefile

```diff
@@ -1,7 +1,7 @@
 TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
 RELEASE_VERSION ?= latest
 
-SUBDIRS=docker-images common-test cluster-operator topic-operator examples
+SUBDIRS=docker-images common-test cluster-operator topic-operator kafka-init examples
```

Comment:
Why is the `kafka-init` not a subdir of `docker-images`?

Reply:
In the `docker-images` folder we have just stuff for building Docker images (Dockerfiles + scripts). The `kafka-init` subproject contains an actual application, it's not just a Docker image as the others.

Reply:
OK, that makes sense. I wonder whether having the directory hierarchy for `docker-images` is really so helpful, or whether a flat structure makes more sense. Right now what governs where something belongs is whether it has any code, which seems somewhat arbitrary. But in any case it's nothing that needs to be fixed for this PR.